packages/react-components/src/components/ChatMessage/ChatMessageComponentAsRichTextEditBox.tsx

```diff
@@ -25,6 +25,10 @@ import { RichTextInputBoxComponent } from '../RichTextEditor/RichTextInputBoxCom
 import { editBoxRichTextEditorStyle, richTextActionButtonsStyle } from '../styles/RichTextEditor.styles';
 import { RichTextSendBoxErrors } from '../RichTextEditor/RichTextSendBoxErrors';
 import { useLocale } from '../../localization';
+/* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
```

Comment:
Could you move this component and edit box one inside `MyMessageComponents` folder